### ISSUE TITLE
I had an error running the usb\device\usb_webcam> instance and I pulled down all the code. There will still be this error, I don't know what went wrong (AEGHB-631)

### ISSUE THREAD

**Issue by garyToo** [espressif/esp-iot-solution#352]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### General issue report

I had an error running the usb\device\usb_webcam> instance and I pulled down all the code. There will still be this error, I don't know where the problem is, the error occurred when I ran the  idf.py set-target esp32s3. This is the error picture. Please help me
![微信截图_20240420140520](https://github.com/espressif/esp-iot-solution/assets/167602698/cff5257b-cb23-4ede-b09f-7879fee52c92)

**Comment by leeebo** [comment]
@garyToo the error caused by`idf-component-manager` `v1.5.*`, which Incorrectly resolved component's dependencies from yml file, you can workaround this by running `pip install "idf-component-manager==1.4.2"` in the IDF environment

**Comment by garyToo** [comment]
Thank you for your guidance. Following your advice, I deleted the build files in usb_webcam. However, I searched the entire esp-iot-solution project and couldn't find the managered_components and dependencies.lock files. I had to proceed with executing the 'idf.py set-target esp32s3' command. But it encountered a new issue 
![ss](https://github.com/espressif/esp-iot-solution/assets/167602698/1b130c21-2b09-41e0-a8f6-0126e6ffb7b1)
